examples/samplers/fast_sampling_with_jax_and_numba.ipynb

```diff
@@ -19,18 +19,58 @@
    "cell_type": "markdown",
```

Comment:
<p>Is this plot needed? You never do anything with the sampling results anyway</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I think just to give the reader an idea of what the data look like. Not needed; just something I did not remove from the current version of the notebook.</p>